src/c++/perf_analyzer/docs/examples/profile.py

```diff
@@ -25,29 +25,77 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import argparse
+import csv
 import json
 import subprocess
-from dataclasses import dataclass
+from dataclasses import asdict, dataclass, fields
 from itertools import pairwise
 from pathlib import Path
 from typing import Optional
 
 import numpy as np
 
 INPUT_FILENAME = "generated_input_data.json"
-
-TITLE = "\n[ BENCHMARK SUMMARY ]\n"
-PROMPT_SIZE = "  Prompt size: {}"
-FIRST_TOKEN_LATENCY = "Average first-token latency: {:.4f} sec"
-T2T_LATENCY = "Average total token-to-token latency: {:.4f} sec"
+METRIC_FIELDS = {
+    "max_first_token_latency": ("Max first token latency", "ms"),
```

Comment:
Just curious, where did the idea for all of these new metrics come from?

Reply:
Some are from me, some from trtllm benchmark script and some from vllm benchmark.